app/cypress/integration/cif/project-revision/index.spec.js

```diff
@@ -94,6 +94,7 @@ describe("the new project page", () => {
     cy.contains("Changes saved").should("be.visible");
     cy.addQuarterlyReport(3, "1993-01-01", "1992-12-31");
     cy.contains("Changes saved").should("be.visible");
+    cy.get('span[class*="MuiChip-label"]').should("have.text", "Complete");
```

Comment:
if this it the best way you can select the status badge, it's probably missing some accessible role (e.g. [status](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/status_role))